### PR TITLE
Material theme beta nav

### PR DESCRIPTION
**Pull request by ContentBrewmaster**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -33,4 +33,153 @@ extra_javascript:
   # menu. Material for MkDocs does not come with JQuery.
   - "https://code.jquery.com/jquery-3.6.2.min.js"
 #nav:
```
Comment (dmundra) [inline mkdocs.yml]: @ContentBrewmaster this needs to be uncommented i.e. remove the '#' symbol.

File: mkdocs.yml [hunk 2]
```diff
@@ -32,5 +32,154 @@ extra_javascript:
   # Read the Docs requires JQuery for its JavaScript code to inject the flyout
   # menu. Material for MkDocs does not come with JQuery.
   - "https://code.jquery.com/jquery-3.6.2.min.js"
-#nav:
-#- Welcome: index.md
+nav:
+- Home: index.md
```
Comment (dmundra) [inline mkdocs.yml]: This should be indented to match `- About CivicActions:`

File: mkdocs.yml [hunk 3]
```diff
@@ -32,5 +32,154 @@ extra_javascript:
...
+  - About CivicActions:
+      - Mission and values: 020-about-us/mission-values.md
+      - Culture: 020-about-us/culture.md
+      - Diversity, Equity, Inclusion, and Accessibility (DEIA):
+          - DEIA at CivicActions: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/purpose.md
+          - Defining DEIA: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/defining-dei.md
+          - Affinity channels: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/affinity-channels.md
+          - DEIA - Get involved: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/deia-get-involved.md
+          - Resources: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/resources.md
```
Comment (dmundra) [inline mkdocs.yml]: Looks like these have an extra `diversity-equity-inclusion` folder in the path

File: mkdocs.yml [hunk 4]
```diff
@@ -32,5 +32,154 @@ extra_javascript:
...
+          - Covid 19 safety: 040-employee-handbook-us/covid19safety.md
+      - Code of conduct: 030-policies/code-of-conduct.md
+      - Workplace guidelines: 030-policies/workplace-guidelines.md
+      - Annual performance review: 030-policies/annual-review-process.md
+      - Performance management: 030-policies/performance-management.md
+      - Leaving Civicactions: 030-policies/leaving-civicactions.md
+      - Health, Safety and Security Policy: 030-policies/health-safety-security.md
+      - New hire orientation:
+          - Welcome to CivicActions!: 010-welcome-to-civicactions/welcome.md
+          - Buddy program: /010-welcome-to-civicactions/training/buddy-program.md
```
Comment (dmundra) [inline mkdocs.yml]: Extra slash at the beginning.

File: mkdocs.yml [hunk 5]
```diff
@@ -32,5 +32,154 @@ extra_javascript:
...
+          - Zoom: 050-how-we-work/tools/zoom.md
+      - Telephone use: 050-how-we-work/tools/telephone.md
+  - Practice areas:
+      - About practice areas: 050-how-we-work/practice-areas/README.md
+      - Accessibility practice area: 050-how-we-work/practice-areas/accessibility-practice-area.md
+      - Design and research:
+          - Design and research methods: 110-ux/services/README.md
+          - Audience and outcomes guide: 110-ux/services/research/audiences-and-outcomes-guide.md
+          - Content audit guide: 110-ux/services/research/content-audit-guide.md
+          - Content modeling guidew: 110-ux/services/design/content-modeling-guide.md
```
Comment (dmundra) [inline mkdocs.yml]: Extra 'w' in 'guidew'

**Review by dmundra — CHANGES_REQUESTED** [review]
Caught some additional typos in the menu.

File: mkdocs.yml (hunk 2 above)
Comment (ContentBrewmaster) [reply mkdocs.yml]: Fixed

File: mkdocs.yml (hunk 3 above)
Comment (ContentBrewmaster) [reply mkdocs.yml]: Fixed

File: mkdocs.yml (hunk 4 above)
Comment (ContentBrewmaster) [reply mkdocs.yml]: Fixed

File: mkdocs.yml (hunk 5 above)
Comment (ContentBrewmaster) [reply mkdocs.yml]: Fixed

File: mkdocs.yml [hunk 6]
```diff
@@ -32,5 +32,164 @@ extra_javascript:
...
+          - Sprint demo: 050-how-we-work/agile-practices/sprint-demo.md
+          - Sprint planning meetings: 050-how-we-work/agile-practices/sprint-planning-meetings.md
+          - Sprint retrospectives: 050-how-we-work/agile-practices/sprint-retrospectives.md
+      - Balance scores: 050-how-we-work/balance-scores.md
+      - Gold Star Program: 050-how-we-work/gold-star-program.md
+      - Security awareness and tools:
+          - Awareness and tools: 100-security/awareness.md
+            - Setting up and using Yubikey: 100-security/yubikey/README.md
+            - Yubikey and Linux: 100-security/yubikey/linux.md
+            - Yubikey and Mac OS: 100-security/yubikey/macosx.md
```
Comment (dmundra) [inline mkdocs.yml]: @ContentBrewmaster I think these 3 should be at the same indentation as `- Awareness and tools: 100-security/awareness.md`

**Review by dmundra — CHANGES_REQUESTED** [review]

File: mkdocs.yml [hunk 7]
```diff
@@ -32,5 +32,164 @@ extra_javascript:
...
+          - Help desk working practices: 120-help-desk/working-practices.md
+      - Project management:
+          - Project manager role: 070-project-management/pm-role.md
+          - Guidelines for starting a new project: 070-project-management/starting-new-project.md
+          - Invoicing: 070-project-management/invoicing.md
+          - Onboarding a new project team member: 070-project-management/onboarding-new-project-team-member.md
+          - When your project gets a contractual Modification (MOD) or Extension: 070-project-management/mods-and-extentions.md
+          - Tracking contract expirations and renewals: 070-project-management/contract-expiration-tracking.md
+          - Tracking contractual requirements: 070-project-management/contractual-requirements.md
+          - Conflict resolution and growth mindset: 070-project-management/growth-mindset/#what-is-growth-mindset
```
Comment (dmundra) [inline mkdocs.yml]: @ContentBrewmaster this link is going to the URL instead of the `.md` version of the file.

File: mkdocs.yml [hunk 8]
```diff
@@ -32,5 +32,164 @@ extra_javascript:
...
+          - Help desk working practices: 120-help-desk/working-practices.md
+      - Project management:
+          - Project manager role: 070-project-management/pm-role.md
+          - Guidelines for starting a new project: 070-project-management/starting-new-project.md
+          - Invoicing: 070-project-management/invoicing.md
+          - Onboarding a new project team member: 070-project-management/onboarding-new-project-team-member.md
+          - When your project gets a contractual Modification (MOD) or Extension: 070-project-management/mods-and-extentions.md
+          - Tracking contract expirations and renewals: 070-project-management/contract-expiration-tracking.md
+          - Tracking contractual requirements: 070-project-management/contractual-requirements.md
+          - Conflict resolution and growth mindset: 070-project-management/growth-mindset/what-is-growth-mindset.md
```
Comment (dmundra) [inline mkdocs.yml]: @ContentBrewmaster I believe `070-project-management/growth-mindset.md` is the file here.

File: mkdocs.yml [hunk 9]
```diff
@@ -32,5 +32,164 @@ extra_javascript:
...
+          - Sprint demo: 050-how-we-work/agile-practices/sprint-demo.md
+          - Sprint planning meetings: 050-how-we-work/agile-practices/sprint-planning-meetings.md
+          - Sprint retrospectives: 050-how-we-work/agile-practices/sprint-retrospectives.md
+      - Balance scores: 050-how-we-work/balance-scores.md
+      - Gold Star Program: 050-how-we-work/gold-star-program.md
+      - Security awareness and tools:
+          - Awareness and tools: 100-security/awareness.md
+            - Setting up and using Yubikey: 100-security/yubikey/README.md
+              - Yubikey and Linux: 100-security/yubikey/linux.md
+              - Yubikey and Mac OS: 100-security/yubikey/macosx.md
```
Comment (dmundra) [inline mkdocs.yml]: @ContentBrewmaster I believe the is causing the error in the checks. I think the identation should look like this:

```
          - Awareness and tools: 100-security/awareness.md
          - Setting up and using Yubikey:
              - 100-security/yubikey/README.md
              - Yubikey and Linux: 100-security/yubikey/linux.md
              - Yubikey and Mac OS: 100-security/yubikey/macosx.md
```

**Review by dmundra — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 9 above)
Comment (dmundra) [reply mkdocs.yml]: @ContentBrewmaster thanks for fixing these. I made one minor change and I think we are good with those.

**Review by dmundra — APPROVED** [review]
Looking good!

**Pull request merged**